.github/renovate.json

```diff
@@ -11,5 +11,13 @@
   },
   "docker": {
     "labels": ["🐳 tech: docker"]
-  }
+  },
+  "ignorePaths": [
+    ".github/workflows/draft_release.yml",
+    ".github/workflows/label_new_pr.yml",
+    ".github/workflows/new_isssues.yml",
+    ".github/workflows/new_prs.yml",
+    ".github/workflows/pr_label_check.yml",
+    ".github/workflows/pr_ping.yml"
```

Comment:
Can we use ".github/workflows/**.yml" here instead?

Reply:
Not all workflows are synced, for example, the `bundle_size.yml` is only in this repo and I guess it would be nice to keep the actions used there updated.